docs/book/installation/sylius_plus_installation.rst

```diff
@@ -291,9 +291,19 @@ Installing Sylius Plus as a plugin to a Sylius application
         use InventorySourceAwareTrait;
     }
 
-**7.** Add wkhtmltopdf binary for Invoicing purposes.
+**7.** Install wkhtmltopdf binary for Invoicing and Refund purposes.
```

Comment:
Not only Invoicing and Refund, because this binary is also used in Plus for GMV or Return Requests